src/LiquidityBalances.js

```diff
@@ -90,21 +91,13 @@ export default function LiquidityBalances(props) {
                       <td className="text-right" rowSpan="2">
                         <Percentage
                           numerator={liquidityToken.balance}
-                          denomiator={liquidityToken.totalSupply}
+                          denominator={liquidityToken.totalSupply}
+                        />
+                        <RewardPerBlock
+                          liquidityToken={liquidityToken}
+                          blockNumber={blockNumber}
+                          MS={MS}
```

Comment:
What do you think about importing MS directly in `RewardPerBlock` since it's a constant?